object/application.go

```diff
@@ -362,3 +362,44 @@ func IsAllowOrigin(origin string) bool {
 
 	return allowOrigin
 }
+
+func GetApplicationsByOrganization(organization string) []*Application {
```

Comment:
This already exists:

![image](https://user-images.githubusercontent.com/3787410/187250011-05db8300-c4cb-4f29-92a6-3cc7f30ad353.png)